### PR TITLE
Bypass canIframeSpikeJump

### PR DESCRIPTION
**Pull request by kjbranch**
Add strats that simply run in the spikes, like a player might do if they didn't know how to utilize the i-frames.

### CODE REVIEW

File: region/norfair/east.json [hunk 1]
```diff
@@ -4645,8 +4650,13 @@
                   "notable": false,
                   "requires": [
                     "h_canNavigateHeatRooms",
+                    "Gravity",
                     {"heatFrames": 375},
-                    {"spikeHits": 2}
+                    {"spikeHits": 2},
+                    {"or": [
+                      "canIframeSpikeJump",
+                      {"spikeHits": 2}
```
Comment (osse101) [inline region/norfair/east.json:4658]: or canDamageBoost tbh

File: region/wreckedship/main.json [hunk 2]
```diff
@@ -1638,6 +1638,13 @@
                     {"spikeHits": 3}
                   ]
                 },
+                {
+                  "name": "Bowling Spike Run",
+                  "notable": false,
+                  "requires": [
+                    {"spikeHits": 7}
```
Comment (osse101) [inline region/wreckedship/main.json:1645]: Looks inconsistent to the other strats.  Should be using no run speed jumps right?

Comment (osse101) [reply region/wreckedship/main.json:1645]: 3 damage boosts would fit in a strat too.

File: region/norfair/east.json (hunk 1 above)
Comment (osse101) [reply region/norfair/east.json:4658]: A note about turning off speed would be nice too while we're here.  I feel it comes up the most in this room.

File: region/norfair/east.json [hunk 3]
```diff
@@ -4585,8 +4585,13 @@
...
```
Comment (osse101) [inline region/norfair/east.json:4593]: or dboost

File: region/crateria/central.json [hunk 4]
```diff
@@ -3358,7 +3358,13 @@
                 {
                   "name": "Take Spike Damage",
                   "notable": false,
-                  "requires": [{"spikeHits": 1}]
+                  "requires": [
+                    {"spikeHits": 1},
+                    {"or": [
+                      "canIframeSpikeJump",
+                      {"spikeHits": 1}
```
Comment (osse101) [inline region/crateria/central.json:3365]: left to right can make it with one damage boost.  Right to left is right on the line, maybe too tight.

File: region/wreckedship/main.json (hunk 2 above)
Comment (kjbranch) [reply region/wreckedship/main.json:1645]: I was doing it as running to the right without jumping. With poor jumps and no run I get 6 hits

**Comment by kjbranch** [comment]
I think damage boosting on the spikes in all of these cases is quite a lot harder than `canIframeSpikeJump`, and kind of still implicitly relies on it. The precision of the damage boost is pretty important to ensure that it even helps reduce the number of hits, and requires quick jumps afterwards, something I would consider part of `canIframeSpikeJump`

**Comment by osse101** [comment]
I can agree with that

**Pull request merged**